frontend/packages/ceph-storage-plugin/src/utils/index.ts

```diff
@@ -0,0 +1,24 @@
+import * as _ from 'lodash';
+import { humanizeBinaryBytesWithoutB } from '@console/internal/components/utils';
+import { PrometheusResponse, DataPoint } from '@console/internal/components/graphs';
+
+export const getMetricType = (resource, metricType) => _.get(resource, ['metric', metricType], '');
+
+export const formatToShortTime = (timestamp: Date): string => {
```

Comment:
you should use `twentyFourHourTime` from `utils/datetime` to be consistent with rest of the console